apps/modernization-ui/src/apps/search/investigation/InvestigationFormTypes.ts

```diff
@@ -30,6 +30,8 @@ type InvestigationFilterEntry = {
     caseStatuses?: Selectable[];
     processingStatuses?: Selectable[];
     notificationStatuses?: Selectable[];
+    reportingProviderId?: Selectable;
```

Comment:
question:  Why add new fields when the `reportingFacility` and `reportingProvider` properties already existed on the object?  Replacing them resulted in extra changes.

Reply:
sorry about that. I created new ones thinking they should be named same as how there expected in the API, I just removed the unused ones and kept the ones I added,